google-startup-scripts/usr/share/google/first-boot

```diff
@@ -29,8 +29,7 @@ function log() {
 }
 
 function get_instance_id() {
-  # Do not retry to get the metadata value on failures.
-  MDS_TRIES=1 ${PREFIX}/usr/share/google/get_metadata_value id 2>/dev/null
+  ${PREFIX}/usr/share/google/get_metadata_value id 2>/dev/null
```

Comment:
In cases where the metadata server may not be reachable, the now default behavior is to retry for 30 seconds. Setting MDS_TRIES=1 when calling get_metadata_value would override this retry and only try once. It doesn't make sense to not retry this metadata check on first-boot anymore than it would make sense in any of the other boot scripts for normal use cases. The great majority of the time, especially with the new check for a connection on boot, a retry would almost never occur. The default retry is therefore to catch any transient failures in getting metadata.